src/Font.cpp

```diff
@@ -627,11 +627,10 @@ void Font::LoadFontPageSettings( FontPageSettings &cfg, IniFile &ini, const RStr
 
 				if(int(wdata.size()) > num_frames_wide)
 				{
-					LuaHelpers::ReportScriptErrorFmt(
-						"The font definition \"%s\" assigns %i characters to row %i"
-						"(\"%ls\"), but the font  is only %i characters wide.",
-						ini.GetPath().c_str(), (int)wdata.size(), row, wdata.c_str(),
-						num_frames_wide);
+					// wstrings don't work. Convert to RString first.
```

Comment:
This may be the biggest issue to worry about. We have the capability to convert `wstring` variables, but what about `wchar_t` characters?